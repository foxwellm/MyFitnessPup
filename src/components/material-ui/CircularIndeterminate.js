import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import CircularProgress from '@material-ui/core/CircularProgress';

const useStyles = makeStyles(theme => ({
  container: {
    display: 'flex',
    justifyContent: 'space-around',
    alignItems: 'center',
    height: 80
  },
  text: {
    fontSize: 20
  },
  progress: {
    margin: theme.spacing(2),
  },

}));

function CircularIndeterminate() {
  const classes = useStyles();

  return (
    <div className={classes.container}>
      <CircularProgress className={classes.progress} />
      <div className={classes.text}>Fetching more dogs that match your criteria</div>
      <CircularProgress className={classes.progress} color="secondary" />
    </div>
  );
}

export default CircularIndeterminate;