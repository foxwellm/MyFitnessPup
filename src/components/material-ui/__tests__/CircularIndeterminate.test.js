import React from 'react'
import { shallow } from 'enzyme'
import CircularIndeterminate from '../CircularIndeterminate'

describe('CircularIndeterminate', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<CircularIndeterminate />)
    expect(wrapper).toMatchSnapshot()
  })
})