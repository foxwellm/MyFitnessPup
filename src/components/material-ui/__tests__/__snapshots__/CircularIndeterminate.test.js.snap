// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularIndeterminate should have match the correct snapshot 1`] = `
<div
  className="makeStyles-container-1"
>
  <WithStyles(ForwardRef(CircularProgress))
    className="makeStyles-progress-3"
  />
  <div
    className="makeStyles-text-2"
  >
    Fetching more dogs that match your criteria
  </div>
  <WithStyles(ForwardRef(CircularProgress))
    className="makeStyles-progress-3"
    color="secondary"
  />
</div>
`;
